src/main/java/com/endava/soa_5/Base.java

```diff
@@ -0,0 +1,14 @@
+package com.endava.soa_5;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Base {
```

Comment:
'Base" is so confusing.. What kind of base? Base for what? Please rename more explicit.

Reply:
Temporarily renamed it, will think with the team members on some more relevant name for this class